src/extensions/mvs/components/primitives.ts

```diff
@@ -110,14 +112,18 @@ export const MVSBuildPrimitiveShape = MVSTransform({
     from: MVSPrimitivesData,
     to: SO.Shape.Provider,
     params: {
-        kind: PD.Text<'mesh' | 'labels' | 'lines'>('mesh')
+        // TODO: some list to not type everything
+        kind: PD.Text<'mesh' | 'labels' | 'lines' | 'box' | 'cylinder'>('mesh')
```

Comment:
```suggestion
        kind: PD.Text<'mesh' | 'labels' | 'lines'>('mesh')
```

These do not correspond to the primitives in MVS, but the primitives Mol* will render.